Increase INWX propagation timeout to 360s to improve robustness

dragetd opened the pull request:
INWX has rather unstable propagation delays. Users reported that the
robustness of the DNS challenge improves with a longer timeout, thus instead
of using the default, hardcode a larger default. Six minutes seems to work
fine so far.

Fixes #1297

dragetd commented:
I was not sure if you want a note in the documentation that the timeout is larger than usual. Tell me if I should change it. :-)

ldez commented on providers/dns/inwx/inwx.go:
```diff
@@ -42,7 +42,8 @@ type Config struct {
 		TTL:                env.GetOrDefaultInt(EnvTTL, 300),
-		PropagationTimeout: env.GetOrDefaultSecond(EnvPropagationTimeout, dns01.DefaultPropagationTimeout),
+		// INWX has rather unstable propagation delays, thus using a larger default value
+		PropagationTimeout: env.GetOrDefaultSecond(EnvPropagationTimeout, 360),
```
```suggestion
		PropagationTimeout: env.GetOrDefaultSecond(EnvPropagationTimeout, 360*time.Second),
```

ldez reviewed (changes requested):
Hello, as your PR comes from a GitHub organization, I'm not able to commit on it.

So could you run `make generate-dns` and commit the result and rebase your PR.

Using a GitHub organization to handle forks is something that can be an issue with bots/automation, I can understand your approach but you have to know the constraints and that can cause problems to open source projects.

dragetd commented:
Uh, I did not know this! How annoying! x.x Well, they added repository-typo filters and by default no longer show forks, it did improve ever since I made this construct. I guess it is time to clean it up!

Sorry for the hassle. :-(

I included your change and the updated documentation, rebased and squashed the changes.

ldez reviewed (approved):
Thanks :+1:

ldez reviewed (changes requested):
Could you run format the source code (gofmt) ? thank you

dragetd commented:
I am sorry for all the hassle :( and I am also nuking my organizations now!
